README.md

```diff
@@ -120,6 +135,25 @@ have explicit support that will be helpful to other developers.
 drush @pantheon.SITENAME.ENV --vendor=pantheon --detail ac
 ````
 
+## Custom Code Paths
+
+Codebase report runs some third party tools on custom code the the drupal site.
+For this, paths containing custom code should be specified while running site_audit.
+This can be done in two ways.
+
+* Provide a comma separated list of paths (files or directories) in `custom-code` option
+ ````
```

Comment:
Should be three, not four quotes.

If you want to be extra good, three quotes followed by the language (bash, php).